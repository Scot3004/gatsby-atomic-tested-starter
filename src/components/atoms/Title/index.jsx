import styled from 'styled-components'

export const Title = styled.h2`
  margin: 0 0 .6rem 0;
  padding: 0;
  word-wrap: none;
`

export default Title
