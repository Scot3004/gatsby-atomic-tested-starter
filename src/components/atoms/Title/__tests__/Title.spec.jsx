import React from 'react'
import 'jest-styled-components'
import renderer from 'react-test-renderer'
import Title from '..'

test('it works', () => {
  const tree = renderer.create(<Title />).toJSON()
  expect(tree).toMatchSnapshot()
})
