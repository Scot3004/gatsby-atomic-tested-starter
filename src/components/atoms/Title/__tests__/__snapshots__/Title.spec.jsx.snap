// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it works 1`] = `
.c0 {
  margin: 0 0 .6rem 0;
  padding: 0;
  word-wrap: none;
}

<h2
  className="c0"
/>
`;
